changelog.d/20240930_104625_aurelien.gateau_ui_refactor.md

```diff
@@ -0,0 +1,3 @@
+### Fixed
+
+- When using the `--debug` option, the log output no longer overlaps with the progress bars.
```

Comment:
is the `--verbose` option also affected ? If so, it could appear in the changelog.

Reply:
It's not affected... yet 😉.

Verbose output already plays well with progress bars. But I plan to make another change so that adding `--debug` implies `--verbose`. At this point I'll probably rework this changelog entry.